.github/workflows/go.yml

```diff
@@ -0,0 +1,31 @@
+name: Go
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+jobs:
+  build:
+    strategy:
+      matrix: 
+        go-versions: [1.12.x, 1.13.x, 1.14.x]
```

Comment:
please add newer go 1.15.x in a follow up PR

Reply:
will get on it right away

Reply:
On a different note ;
- Travis CI was explicit about distro and release (ubuntu-xenial) and the github-actions is now using "ubuntu-latest" which is translating to xenial for now. But the GH-Actions is also showing this message ;

```
Ubuntu-latest workflows will use Ubuntu-20.04 soon. For more details, see https://github.com/actions/virtual-environments/issues/1816
```
Does this require any attention or is "ubuntu-latest" ok when GH starts using focal

Reply:
Also, please advise if "only" add 1.15.x or remove 1.12.x as well. thnx